homeassistant/components/airtouch4/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for AirTouch4."""
+from airtouch4pyapi import AirTouch
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_HOST
+
+from .const import DOMAIN
+
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
+
+
+def _createAirtouchObject(host):
+    return AirTouch(host)
+
+
+async def _validate_connection(hass: core.HomeAssistant, host):
+    airtouch = await hass.async_add_executor_job(_createAirtouchObject, host)
+
+    if hasattr(airtouch, "error"):
+        return airtouch.error
+    return bool(airtouch.GetGroups())
+
+
+class AirtouchConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle an Airtouch config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    @core.callback
+    def _async_get_entry(self, data):
+
+        return self.async_create_entry(
+            title=data[CONF_HOST],
+            data={
+                CONF_HOST: data[CONF_HOST],
+            },
+        )
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        if user_input is None:
+            return self.async_show_form(step_id="user", data_schema=DATA_SCHEMA)
+
+        errors = {}
+
+        host = user_input[CONF_HOST]
+
+        await self.async_set_unique_id(user_input[CONF_HOST])
```

Comment:
Please check our criteria for unique id:
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-id-requirements

Reply:
That same criteria seems to indicate (unless I'm wrong) that the unique id is not required at all if not using discovery: "If a unique ID isn't available, alternatively, the dhcp, zeroconf, hassio, homekit, ssdp and discovery steps can be omitted, even if they are configured in the integration manifest. In that case, the user step will be called when the item is discovered." - since I don't have access to a suitable piece of information, will omit entirely for now